types/jenkins/index.d.ts

```diff
@@ -1,162 +1,70 @@
-// Type definitions for jenkins 0.23
+// Type definitions for jenkins 1.0.2
 // Project: https://github.com/silas/node-jenkins
 // Definitions by: rhysd <https://rhysd.github.io>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
+interface JenkinsClientOptions {
+    baseUrl: string;
+    crumbIssuer?: boolean | undefined | `() => void`;
```

Comment:
I don't think this string is what you intended to use here.

Reply:
Fixed to clearly define that it is a function definition.
[`30d12bd` (#66784)](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66784/commits/30d12bdb91b136277ca1580500460808db32e2c1)